manifests/instance.pp

```diff
@@ -356,7 +359,11 @@
       $real_service_ensure = $service_ensure == 'running'
       $real_service_enable = $service_enable
 
-      Exec["cp -p ${redis_file_name_orig} ${redis_file_name}"] ~> Service["${service_name}.service"]
+      if $notify_service {
+        Exec["copy ${redis_file_name_orig} to ${redis_file_name}"] ~> Service["${service_name}.service"]
+      } else {
+        Exec["copy ${redis_file_name_orig} to ${redis_file_name}"] -> Service["${service_name}.service"]
+      }
```

Comment:
Is this needed?

Reply:
I think it still makes sense to set up this dependency chaining for config file to exist prior starting the instance.